server/services/licenceService.js

```diff
@@ -32,7 +31,7 @@ module.exports = function createLicenceService(licenceClient, establishmentsClie
             const status = getIn(rawLicence, ['status']);
 
             if(populateConditions && !isEmpty(formattedLicence.additionalConditions)) {
-                return await populateLicenceWithConditions(formattedLicence, status);
+                return populateLicenceWithConditions(formattedLicence, status);
```

Comment:
The function no longer needs to be async anymore so we can get rid of the `try catch` and `async` 😃